samples/pom.xml

```diff
@@ -29,7 +29,7 @@
         <module>spring-boot-no-feature-provider-or-feature-enums</module>
         <module>spring-boot-security</module>
         <module>spring-boot-thymeleaf</module>
-        <module>spring-boot-war</module>
+        <!--<module>spring-boot-war</module> TODO: Fix war sample -->
```

Comment:
I've fixed spring-boot-war by defining
```
                <plugin>
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-war-plugin</artifactId>
                    <version>3.3.2</version>
                </plugin>
```
in `pluginManagement` section of `\samples\pom.xml`

After that it became useless to define explicitly the version of this plugin in `\samples\spring-boot-hello-world\pom.xml`

Reply:
Great, thank you!

Reply:
I suggested to remove just version number from `\samples\spring-boot-hello-world\pom.xml` keeping the plugin itself with its settings without changes. Does it work actually when the whole plugin reference removed?